src/promptflow-devkit/promptflow/_sdk/entities/_flows/flex.py

```diff
@@ -33,9 +34,20 @@ def __init__(
         code = Path(code)
         # entry function name
         self.entry = entry
+        self._flow_dir, self._flow_file_name = resolve_flow_path(path,
+                                                                 check_flow_exist=False,
+                                                                 default_flow_file=FLOW_FLEX_YAML)
```

Comment:
seems that `check_flow_exist` and `default_flow_file` are highly bonded? `default_flow_file` will and only will take effect when `check_flow_exist` is set to `False`?

Reply:
Yes